src/Ormolu/Printer/Meat/Type.hs

```diff
@@ -269,22 +267,6 @@ p_conDeclField ConDeclField {..} = do
   breakpoint
   sitcc . inci $ p_hsType (unLoc cd_fld_type)
 
-tyOpTree :: LHsType GhcPs -> OpTree (LHsType GhcPs) (Located RdrName)
```

Comment:
I moved `tyOpTree` and `p_tyOpTree` to the `Value.hs`, because they have a lot of symmetry with `cmdOpTree`, `exprOpTree` and `p_cmdOpTree`, `p_exprOpTree` and also need to reuse some helper functions defined but not exposed by `Value.hs`. If this is not acceptable, maybe we could create an `OpTree.hs` file?

Reply:
Since `OpTree` is used for both types and values, I think it makes sense to keep it in a separate file, and not in `Type.hs` or `Value.hs`.

Reply:
I moved them to `OpTree.hs`. I suppose we can resolve this conversation thread?